simple_certmanager/models.py

```diff
@@ -3,11 +3,16 @@
 from django.db import models
 from django.utils.translation import gettext, gettext_lazy as _
 
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
```

Comment:
can you move the actual `cryptography` usage into a separate module `simple_certmanager/csr_generation.py` and import the functions from there? I'd like to separate the concerns in `models.py` and apply loose coupling.

Reply:
please address this feedback :)